### DISCUSSION TITLE
Missing visual clues on new docs

### DISCUSSION THREAD

**Discussion by yash-p-amd** [rails/rails#51379] [Docs]
New docs are looking clean and easy to read although I missed some of the things from the legacy design. For example green checkmarks in the "After reading this guide" section. It clearly shows what would I achieve after following the guide and it makes navigation easier. I think we shouldn't hesitate to use emojis when needed.

**Comment by AmandaPerino** [comment]
Hi @yash-p-amd Am also a fan of emojis and iconsl but we did discuss this and decided against checkmarks in favor of bulletpoints to make the design on first glance (top of page) much cleaner. Also because checkmarks without being able to check/uncheck them doesn't quite feel right. But we are open to other suggestions besides checkmarks and bulletpoints. Will look into what those options could be.

Thanks for your suggestion!

**Comment by AmandaPerino** [comment]
Hi again! We added a visual cue after the intro - a line divider in place of the blue. Updated in #51444. Thanks again for the feedback!
